src/NLog.Extensions.Logging/Logging/NLogProviderOptions.cs

```diff
@@ -56,6 +56,18 @@ public class NLogProviderOptions
 #endif
         public bool IncludeActivtyIdsWithBeginScope { get; set; }
 
+        /// <summary>
+        /// Resets the default Microsoft LoggerFactory Filter for the <see cref="NLogLoggerProvider"/>
```

Comment:
```suggestion
        /// Removes the default Microsoft LoggerFactory Filter for the <see cref="NLogLoggerProvider"/>
```

Reply:
Well I'm not clever enough to read from documentation what [AddFilter](https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.filterloggingbuilderextensions.addfilter?view=dotnet-plat-ext-5.0#Microsoft_Extensions_Logging_FilterLoggingBuilderExtensions_AddFilter_Microsoft_Extensions_Logging_LoggerFilterOptions_System_String_System_Func_Microsoft_Extensions_Logging_LogLevel_System_Boolean__) does when receiving null as input-parameter.